controls/pcidss_4_ocp4.yml

```diff
@@ -3036,15 +3080,24 @@ controls:
         - Any changes to time settings on critical systems are logged, monitored, and reviewed.
       levels:
         - base
-      status: pending
+      status: planned
+      notes: |-
+        As NTP is configured directly on the node the audit should also be configured there.
       rules: []
+      related_rules:
+        - audit_rules_time_watch_localtime
+        - audit_rules_time_settimeofday
+        - audit_rules_time_clock_settime
+        - audit_rules_time_stime
+        - audit_rules_time_adjtimex
+        - chronyd_run_as_chrony_user
```

Comment:
Those rules are also from the Linux group.

Reply:
Similar comment as above. I think Yuuma is just putting these here so that we can bootstrap them in an RHCOS4 profile eventually, without affecting the actual OpenShift profiles.

Reply:
Yes, :grinning: 
As I went through the 3.2.1 profile and searched through the rules, I added notable rules that can be useful for RHCOS 4 profile.